data/src/main/java/com/globant/equattrocchio/data/RealmDB.java

```diff
@@ -0,0 +1,34 @@
+package com.globant.equattrocchio.data;
+
+import com.globant.equattrocchio.data.entities.ImageEntity;
+import com.globant.equattrocchio.domain.enities.Image;
+
+import java.util.List;
+
+import io.realm.Realm;
+
+
+public class RealmDB {
```

Comment:
Rename to something more propertly. for example ImageReporsitory

Reply:
ok done